README.md

```diff
@@ -107,6 +109,38 @@ If libzim is compiled without Xapian, all search API are removed.  You
 can test if an installed version of libzim is compiled with or without
 xapian by testing the define `LIBZIM_WITH_XAPIAN`.
 
+Testing
+-------
+
+The unit test need testing zim files.
```

Comment:
Change this sentence to "ZIM files needed by unit-tests are not included in this repository. You must tell meson where the test data is located."